src/server/rdb_test.cc

```diff
@@ -33,33 +32,18 @@ using absl::StrCat;
 ABSL_DECLARE_FLAG(int32, list_compress_depth);
 ABSL_DECLARE_FLAG(int32, list_max_listpack_size);
 ABSL_DECLARE_FLAG(int, compression_mode);
-ABSL_DECLARE_FLAG(string, dbfilename);
 
 namespace dfly {
 
 class RdbTest : public BaseFamilyTest {
  protected:
-  void TearDown();
   void SetUp();
 
   io::FileSource GetSource(string name);
 };
 
 void RdbTest::SetUp() {
-  SetFlag(&FLAGS_dbfilename, "rdbtestdump");
-  BaseFamilyTest::SetUp();
-}
-
-void RdbTest::TearDown() {
-  // Disable save on shutdown
-  SetFlag(&FLAGS_dbfilename, "");
-
-  auto rdb_files = io::StatFiles("rdbtestdump*");
-  CHECK(rdb_files);
-  for (const auto& fl : *rdb_files) {
-    unlink(fl.name.c_str());
-  }
-  BaseFamilyTest::TearDown();
+  InitWithDbFilename();
```

Comment:
Ok, I understand now the context but I think for these tests `SetFlag(&FLAGS_dbfilename, "rdbtestdump");` is enough.

Reply:
Even if not, that's a good infra to have, which already semi exists for `rdb_test`, I think it's better to keep it for future use as well